package.json

```diff
@@ -14,6 +14,7 @@
   },
   "license": "MIT",
   "dependencies": {
+    "highlightjs-badge": "^0.1.9",
```

Comment:
can we fix these in place. i know a lot of these are prior to your work but they should not be ^